content/projects/gsoc/index.adoc

```diff
@@ -26,54 +26,79 @@ in completing their summer projects.
+Projects may also have their own mailing lists, chats and meetings.
+See details on project pages.
+
+=== Mailing lists
 
-=== Contacts
+Public Google Group: link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[jenkinsci-gsoc-all-public].
 
-* Public Google Group: https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public
 1. To start, you need a Google or Gmail account
 2. After clicking above link, a "Sign in" button appears in top-right of page
 3. After sign in, a "Join group" button appears in middle of page
 4. To learn more about joining a Google Group, please see https://support.google.com/groups/answer/1067205
 
-* Chats:
-** link:https://gitter.im/jenkinsci/gsoc-sig[GSoC SIG room in Gitter] for organizational topics
-** `#jenkins` on Freenode IRC network for technical topics. (link:/chat/[More info])
-** each project also has its own Gitter chat, see project pages
+=== Chats
 
-=== Office hours
+** link:https://gitter.im/jenkinsci/gsoc-sig[GSoC Gitter channel] for organizational topics
+** link:/chat/[Common developer chats] for technical topics.
 
-There is a weekly GSoC SIG project meeting,
-which happens at 2PM UTC on Wednesdays.
-The meetings are usually recorded,
-and we post links in link:https://gitter.im/jenkinsci/gsoc-sig[our gitter channel]
-before the meeting starts.
-You can find recording of office hours link:https://www.youtube.com/playlist?list=PLN7ajX_VdyaO1f6bvkcSzW4PdWKkLktRG[here].
+=== Office hours
 
 Although we use mailing lists as the main communication channel,
-we also define regular timeslots aka "office hours".
-During these timeslots Jenkins GSoC org admins and mentors will be also available
-in the Gitter channel.
-Video calls will be conducted on-demand.
+we also have regular "office hours" video calls.
+During these timeslots Jenkins GSoC org admins and mentors are available for any GSoC-related questions.
+
+Before the announcement of accepted projects,
+we will be using a weekly link:sigs/gsoc[GSoC SIG] meeting as the office hours slot (2PM UTC on Wednesdays).
+More slots may be added in the future.
+We post links in link:https://gitter.im/jenkinsci/gsoc-sig[our Gitter channel]
+before the meeting starts.
+Meetings are recorded on-demand and posted link:https://www.youtube.com/playlist?list=PLN7ajX_VdyaO1f6bvkcSzW4PdWKkLktRG[here].
 
-In addition to these meetings,
+In addition to these organization-wide meetings,
 each GSoC project has regular meetings during community bonding and coding phases.
 See the project pages for the schedule.
 
+== Org admins
+
+GSoC Organization admins in 2019:
+link:https://github.com/martinda[Martin d'Anjou],
+link:https://github.com/jeffpearce[Jeff Pearce],
+link:https://github.com/lloydchang[Lloyd Chang],
+link:https://github.com/oleg-nenashev/[Oleg Nenashev].
+
+Jenkins Org Admins mailing list: jenkins-gsoc-2019-org-admins@googlegroups.com.
+Use this mailing list if you need to discuss something privately with org admins.
```

Comment:
This mailing list is not for the application process, nor for technical project questions or discussions. It is only for personal matters, for example communication difficulties with a student or a mentor, or when a mentor or a student are no longer available.